benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/allkernels_tasks.cl

```diff
@@ -0,0 +1,482 @@
+
+// LEN value predefined for a problem size of 250x250
+#define LEN 256
+
+__attribute__((vec_type_hint(double)))
+__attribute__ ((reqd_work_group_size(1, 1, 1)))
+__attribute__((xcl_zero_global_work_offset))
```

Comment:
Is this a Xilinx specific setting?

Reply:
Yes it is, but unrecognized __attributes__ are just ignored by other backends. It still works on CPU and GPU.